src/Native/build.sh

```diff
@@ -103,6 +103,23 @@ fi
 
 __cmake_defines="${__cmake_defines} -DVERSION_FILE_PATH:STRING=${__versionSourceFile}"
 
+OS_ARCH=$(uname -m)
+OS=$(uname)
+
+# If we are cross compiling on Linux we need to set the CMAKE_TOOLCHAIN_FILE
+if [[ ( $OS_ARCH == "amd64" || $OS_ARCH == "x86_64" ) && ( $__build_arch == "arm64" || $__build_arch == "arm" ) && $OS != "Darwin" ]] ; then
+    __cmake_defines="${__cmake_defines} -DCMAKE_TOOLCHAIN_FILE=$(pwd)/../../eng/common/cross/toolchain.cmake"
```

Comment:
You can use `$RootRepo/eng/common/...` here instead of assuming where this file exists in the source tree.

Reply:
Thats good to know. I had tried `$RepoRoot` and that didn't work so I assumed there wasn't anything for it. Code has been changed.